packages/notify-client/src/controllers/engine.ts

```diff
@@ -1809,4 +1856,87 @@ export class NotifyEngine extends INotifyEngine {
       this.finishedInitialLoad = true;
     }
   };
+
+  private readNotifications = async ({
+    topic,
+    notificationIds,
+    all,
+  }: {
+    topic: string;
+    notificationIds: string[] | null;
+    all: boolean;
+  }): Promise<void> => {
+    this.isInitialized();
+
+    if (!this.client.subscriptions.keys.includes(topic)) {
+      throw new Error(`No subscription with topic ${topic} exists`);
+    }
+
+    const subscription = this.client.subscriptions.get(topic);
+
+    const identityKey = encodeEd25519Key(
+      await this.client.identityKeys.getIdentity({
+        account: subscription.account,
+      })
+    );
+
+    const issuedAt = Math.round(Date.now() / 1000);
```

Comment:
```suggestion
    const issuedAtInSeconds = Math.round(Date.now() / 1000);
```

The original name `issuedAt` is descriptive for someone familiar with the context in which it's used, but it doesn't explicitly explain the unit of time being used. Using `issuedAtInSeconds` as the variable name clearly indicates that the time is measured in seconds.